### DISCUSSION TITLE
Migration from 1.3 to 1.4

### DISCUSSION THREAD

**Discussion by albertkurian90** [sqlalchemy/sqlalchemy#11996] [Usage Questions]
I am doing a migration from 1.3.22 to 1.4.49. I have 2 model classes inheriting declarative_base from sqlalchemy.ext.declarative.declarative_base. class Parent and class Child. 
In class Child I have a "one to many" relationship defined with the parent. 
When I startup the application I get an error: "AttributeError: entity" also "failed to initialize one or more mappers" with the name of my Parent class. 
This was a working application with 1.3, any ideas are appreciated. Thank you

**Comment by CaselIT** [comment]
Hi,

Not much to go by. Can you provide a complete stacktrace and the models in question?
